Makefile

```diff
@@ -214,11 +204,8 @@ build: shell_target := --target=dev
 else
 build: shell_target := --target=final
 endif
-ifdef USE_BUILDX
-build: buildx_load := --load
-endif
 build: buildx
-	$(BUILD_CMD) $(BUILD_OPTS) $(shell_target) $(buildx_load) $(BUILD_CROSS) -t "$(DOCKER_IMAGE)" .
+	$(BUILD_CMD) $(BUILD_OPTS) $(shell_target) --load $(BUILD_CROSS) -t "$(DOCKER_IMAGE)" .
```

Comment:
Will the need for `--load` (and custom builders) go away once we have the containerd integration? Perhaps we need a variable to allow switching once that arrives.

Reply:
Yes but the switch should be set in follow-up in https://github.com/rumpl/moby/ when rebased with this PR.